smart-contracts/contracts/BridgeBank/BridgeToken.sol

```diff
@@ -12,13 +12,15 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 contract BridgeToken is ERC20Burnable, Ownable {
```

Comment:
Let's add a BridgeToken interface here that has all of the public functions on it so that the bridgebank can understand how to pull the cosmosDenom.